### PR TITLE
remove depends block in fmj

### PR DESCRIPTION
**Pull request by triphora**
It works just fine on 1.16 with zero modifications

### CODE REVIEW

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -20,12 +20,5 @@
   },
   "mixins": [
     "pride-year.mixins.json"
-  ],
-
-  "depends": {
-    "fabricloader": ">=0.11.3",
-    "minecraft": "1.17.x",
-    "java": ">=16",
-    "quilt-loading-screen": "2.x"
```
Comment (TehcJS) [inline src/main/resources/fabric.mod.json:29]: the mod depends on quilt-loading-screen 2.x, so we can remove the `minecraft` dependency. also, it does require java 16 to run :/, so we can't do much more than that.

Comment (sylv256) [inline src/main/resources/fabric.mod.json:29]: the mod depends on quilt-loading-screen 2.x, so we can remove the `minecraft` dependency. also, it does require java 16 to run :/, so we can't do much more than that.

Comment (sylv256) [reply src/main/resources/fabric.mod.json:29]: also we can remove `fabricloader >=0.11.3` since it doesn't use any fabricloader stuff

**Review by sylv256 — CHANGES_REQUESTED** [review]
keep `java: >=16` and `quilt-loading-screen: 2.x`

File: src/main/resources/fabric.mod.json (hunk 1 above)
Comment (triphora) [reply src/main/resources/fabric.mod.json:29]: > the mod depends on quilt-loading-screen 2.x

no it doesn't, it works fine on 1.x

> it does require java 16 to run :/

no it doesn't

**Comment by triphora** [comment]
config option in upstream now so this doesn't matter anymore

**Pull request closed without merging**